docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -550,60 +550,123 @@ The default is 30303. Ports must be [exposed appropriately](../Configuring-Panth
 
-The default is `false`. 
-    
-### permissions-config-file    
+### permissions-accounts-config-file    
 
 ```bash tab="Syntax"
---permissions-config-file=<FILE>
+--permissions-accounts-config-file=<FILE>
 ```
 
 ```bash tab="Example Command Line"
---permissions-config-file=/home/me/me_configFiles/myPermissionsFile
+--permissions-accounts-config-file=/home/me/me_configFiles/myPermissionsFile
 ```
 
 ```bash tab="Example Configuration File"
-permissions-config-file="/home/me/me_configFiles/myPermissionsFile"
+permissions-accounts-config-file="/home/me/me_configFiles/myPermissionsFile"
 ```
 
-Path to the [permissions configuration file](../Permissions/Permissioning.md#permissions-configuration-file).
-The default is the `permissions_config.toml` file in the [data directory](#data-path).
+Path to the [accounts permissions configuration file](../Permissions/Permissioning.md#permissions-configuration-file).
+Default is the `permissions_config.toml` file in the [data directory](#data-path).
+
+!!! tip
+    `--permissions-accounts-config-file` and [`--permissions-nodes-config-file`](#permissions-nodes-config-file)
+    can use the same file. 
 
 !!!note
     This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#permissions-configuration-file).
 
-### permissions-nodes-enabled
+### permissions-nodes-config-file-enabled
 
 ```bash tab="Syntax"
---permissions-nodes-enabled[=<true|false>]
+--permissions-nodes-config-file-enabled[=<true|false>]
 ```
 
 ```bash tab="Example Command Line"
---permissions-nodes-enabled
+--permissions-nodes-config-file-enabled
 ```
 
 ```bash tab="Example Configuration File"
-permissions-nodes-enabled=true
+permissions-nodes-config-file-enabled=true
 ```
 
-Set to enable node level permissions.
-The default is `false`.
+Set to enable file-based node level permissions. Default is `false`.
+
+### permissions-nodes-config-file    
+
+```bash tab="Syntax"
+--permissions-nodes-config-file=<FILE>
+```
+
+```bash tab="Example Command Line"
+--permissions-nodes-config-file=/home/me/me_configFiles/myPermissionsFile
+```
+
+```bash tab="Example Configuration File"
+permissions-nodes-config-file="/home/me/me_configFiles/myPermissionsFile"
+```
+
+Path to the [nodes permissions configuration file](../Permissions/Permissioning.md#permissions-configuration-file).
+Default is the `permissions_config.toml` file in the [data directory](#data-path).
+
+!!! tip
+    `--permissions-nodes-config-file` and [`--permissions-accounts-config-file`](#permissions-accounts-config-file)
+    can use the same file. 
+
+!!!note
+    This option is not used when running Pantheon from the [Docker image](../Getting-Started/Run-Docker-Image.md#permissions-configuration-file).
+
+### permissions-nodes-contract-address
+
+```bash tab="Syntax"
+--permissions-nodes-contract-address=<ContractAddress>
+```
+
+```bash tab="Example Command Line"
+--permissions-nodes-contract-address=xyz
```

Comment:
shouldn't we have a valid address that would also show if we have to use the 0x prefix ?

Reply:
Yes - I'll add this when I'm doc'ing the functionality (not available yet).  Adding a note to the JI so I don't forget.